src/pages/home/HeaderView.js

```diff
@@ -172,12 +173,14 @@ function HeaderView(props) {
     const shouldShowBorderBottom = !isTaskReport || !props.isSmallScreenWidth;
     const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(props.report);
 
+    const isLoading = !(props.report && title);
```

Comment:
```suggestion
    const isLoading = !props.report || !title;
```

Actually, there's no case of `!props.report`. `!title` should be enough condition